### PR TITLE
Add Kubermatic to k8s-infra-staging-apisnoop project

### PR DESCRIPTION
**Pull request by koksay**
Access to `k8s-infra-staging-apisnoop` project to work on apisnoop and conformance tests.

Fyi, @xmudrii @mfahlandt @jeefy 

/cc @BobyMCbobs

### CODE REVIEW

**Comment by BobyMCbobs** [comment]
Thank you @koksay, these are the perms we need.

Seeing how I can help below
/lgtm
/approve

File: groups/sig-architecture/groups.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ groups:
       - hh@ii.coop
       - jbelamaric@google.com
       - zz@ii.coop
+      - koray@kubermatic.com
+      - mario@kubermatic.com
+      - marko@kubermatic.com
```
Comment (xmudrii) [inline groups/sig-architecture/groups.yaml:73]: Can you please use `mudrinic.mare@gmail.com` as I use it for everything else in the upstream projects?

Comment (koksay) [reply groups/sig-architecture/groups.yaml:73]: right, maybe I should use my gmail one as well for the same reason. @mfahlandt do you have any preference?

**Pull request merged**